src/main/resources/anniv5Resources/localization/eng/cthulhupack/Cardstrings.json

```
@@ -1,33 +1,33 @@
 {
   "${ModID}:BeyondTheStars": {
       "NAME": "Beyond the Stars",
-    "DESCRIPTION": "Deal !D! damage to ALL enemies. NL Gain a *Madness.",
-    "UPGRADE_DESCRIPTION": "Deal !D! damage to ALL enemies. NL Gain a *Madness+."
+    "DESCRIPTION": "Deal !D! damage to ALL enemies. NL Lose !M! ${ModID}:Sanity for each enemy hit."
   },
   "${ModID}:NamelessMist": {
     "NAME": "Nameless Mist",
-    "DESCRIPTION": "Gain !B! Block. NL ${ModID}:Flurry: Gain !M! Block."
+    "DESCRIPTION": "Gain !B! Block. NL ${ModID}:Flurry: Gain !M! Temporary_HP."
   },
   "${ModID}:FlailingTendril": {
     "NAME": "Flailing Tendril",
-    "DESCRIPTION": "Deal !D! damage and apply !M! ${ModID}:Distortion to a random enemy."
+    "DESCRIPTION": "Deal !D! damage to a random enemy. NL Lose !M! ${ModID}:Sanity. NL If this costs 0, it hits twice."
   },
   "${ModID}:PageOfTheDead": {
     "NAME": "Page of the Dead",
-    "DESCRIPTION": "Gain a *Necronomicurse. NL The next Attack you play that costs 2 or more is played twice.",
-    "UPGRADE_DESCRIPTION": "The next Attack you play that costs 2 or more is played twice."
+    "DESCRIPTION": "Lose !M! ${ModID}:Sanity. NL Add a random Attack that costs 2 to your hand.",
+    "UPGRADE_DESCRIPTION": "Lose !M! ${ModID}:Sanity. NL Add a random *Upgraded Attack that costs 2 to your hand."
   },
   "${ModID}:FamilialCurse": {
     "NAME": "Familial Curse",
-    "DESCRIPTION": "Apply !M! ${ModID}:Ruin to ALL enemies. NL Next turn, gain a *Madness and lose 1 HP."
+    "DESCRIPTION": "Gain !B! Block. NL Apply !M! Weak to ALL enemies. NL Lose 4 ${ModID}:Sanity."
   },
   "${ModID}:GazeTheVoid": {
     "NAME": "Gaze the Void",
-    "DESCRIPTION": "Deal !D! damage !M! times. NL Enter ${ModID}:Nightmare."
+    "DESCRIPTION": "Deal !D! damage to a random enemy. NL Repeat for each *Lunacy played this combat.",
+    "EXTENDED_DESCRIPTION": [" NL (Hits ", " times.)"]
   },
   "${ModID}:AllSeeing": {
-    "NAME": "All Seeing",
-    "DESCRIPTION": "At the end of each turn, Scry !M! and take 3 damage."
+    "NAME": "Eldritch Pact",
+    "DESCRIPTION": "At the start of each turn, lose !M! ${ModID}:Sanity. NL Whenever you lose *Sanity, gain !M! Block."
```

A minor change to use `${ModID}:Sanity` instead of `*Sanity`. I'll just make it through Github.
```suggestion
    "DESCRIPTION": "At the start of each turn, lose !M! ${ModID}:Sanity. NL Whenever you lose ${ModID}:Sanity, gain !M! Block."
```